client/state/login/selectors.js

```diff
@@ -152,7 +146,7 @@ export const getRequestError = ( state ) => {
  * @returns {?object}         Notice for the request.
  */
 export const getRequestNotice = ( state ) => {
-	return get( state, 'login.requestNotice', null );
+	return state.login.requestNotice ?? null;
```

Comment:
Similar case as with the error states: `requestNotice` is either `null` or a valid object, except if `action.notice` was undefined in relevant actions.

Reply:
Done.